persistence/relational-jdbc/src/main/java/org/apache/polaris/persistence/relational/jdbc/DatasourceOperations.java

```diff
@@ -116,6 +116,28 @@ public void executeScript(InputStream scriptInputStream) throws SQLException {
     }
   }
 
+  public boolean checkSchemaExists(@Nonnull String schemaName) {
+    final String sql = "SELECT 1 FROM information_schema.schemata WHERE schema_name = ?";
```

Comment:
Is `information_schema.schemata` standard? :thinking: Could it be preferable to use JDBC APIs (`Connection.getMetaData()`) for  this and let the driver figure out the right source for this data?

Reply:
PostgreSQL is the only currently supported database. I would not want Polaris to be limited to PostgreSQL forever :)